src/DssSpell.sol

```diff
@@ -75,334 +60,240 @@ contract DssSpellAction is DssAction {
-        // Forum: https://forum.sky.money/t/2024-10-17-expected-executive-contents-rwa-vault-changes/25323
-
-        // Remove RWA007-A from Debt Ceiling Instant Access Module
-        DssExecLib.removeIlkFromAutoLine("RWA007-A");
-
-        // Note: in order to decrease global debt ceiling, we need to fetch current `line`
-        (,,, uint256 line1,) = VatAbstract(MCD_VAT).ilks("RWA007-A");
-
-        // Set RWA007-A Debt Ceiling to 0
-        DssExecLib.setIlkDebtCeiling("RWA007-A", 0);
-
-        // Initiate RWA007-A soft liquidation by calling `tell()`
-        RwaLiquidationOracleLike(MIP21_LIQUIDATION_ORACLE).tell("RWA007-A");
-
-        // Write-off the debt of RWA007-A and set its oracle price to 0 by calling `cull()`
-        RwaLiquidationOracleLike(MIP21_LIQUIDATION_ORACLE).cull("RWA007-A", RWA007_A_URN);
-
-        // Note: update the spot value in vat by propagating the price
-        DssExecLib.updateCollateralPrice("RWA007-A");
-
-        // Note: in order to decrease global debt ceiling, we need to fetch current `line`
-        (,,, uint256 line2,) = VatAbstract(MCD_VAT).ilks("RWA014-A");
-
-        // Reduce RWA014-A Debt Ceiling by 1.5 billion Dai from 1.5 billion Dai to 0
-        DssExecLib.setIlkDebtCeiling("RWA014-A", 0);
-
-        // Initiate RWA014-A soft liquidation by calling `tell()`
-        RwaLiquidationOracleLike(MIP21_LIQUIDATION_ORACLE).tell("RWA014-A");
-
-        // Write-off the debt of RWA014-A and set its oracle price to 0 by calling `cull()`
-        RwaLiquidationOracleLike(MIP21_LIQUIDATION_ORACLE).cull("RWA014-A", RWA014_A_URN);
-
-        // Note: update the spot value in vat by propagating the price
-        DssExecLib.updateCollateralPrice("RWA014-A");
-
-        // Note: decrease global line
-        VatAbstract(MCD_VAT).file("Line", VatAbstract(MCD_VAT).Line() - (line1 + line2));
-
-        // ---------- Pinwheel DAO Resolution ----------
-        // Forum: https://forum.sky.money/t/coinbase-web3-wallet-legal-overview/24577/3
-
-        // Approve DAO Resolution at QmYJUvw5xbAJmJknG2xUKDLe424JSTWQQhbJCnucRRjUv7
-        // Note: see `dao_resolutions` public variable declared above
-
-        // ---------- AAVE Revenue Share Payment ----------
-        // Forum: https://forum.sky.money/t/spark-aave-revenue-share-calculation-payment-5-q3-2024/25286
-
-        // AAVE Revenue Share - 234089 DAI - 0x464C71f6c2F760DdA6093dCB91C24c39e5d6e18c
-        DssExecLib.sendPaymentFromSurplusBuffer(AAVE_V3_TREASURY, 234_089);
-
-        // ---------- Spark Spell ----------
-        // Forum: https://forum.sky.money/t/oct-3-2024-proposed-changes-to-spark-for-upcoming-spell/25293
-        // Poll: https://vote.makerdao.com/polling/QmbHaA2G
-        // Poll: https://vote.makerdao.com/polling/QmShWccA
-        // Poll: https://vote.makerdao.com/polling/QmTksxrr
+        // ---------- Init Base Native Bridge ----------
+        // Forum: https://forum.sky.money/t/spell-contents-2024-10-31/25421/
+        //
```

Comment:
`Init Base Token Bridge for USDS, sUSDS by calling TokenBridgeInit.initBridges using the following parameters:` is missing

in the previous spell this comment was added with the following comment.
`Note: this is a subheading, actual instructions are below`